pkg/api/types.go

```diff
@@ -293,6 +297,13 @@ type TestStepConfiguration struct {
 	OpenshiftInstallerSrcClusterTestConfiguration   *OpenshiftInstallerSrcClusterTestConfiguration   `json:"openshift_installer_src,omitempty"`
 }
 
+// Secret describes a secret to be mounted inside a test
+// container.
+type Secret struct {
+	Name      string `json:"name"`
```

Comment:
This is where people will go after the README for docs on the fields, so let's make sure they are documented here